app/cronjob/trailingTrade/step/handle-open-orders.js

```diff
@@ -153,7 +153,7 @@ const execute = async (logger, rawData) => {
           data.accountInfo = await updateAccountInfo(
             logger,
             balances,
-            moment().utc().format()
+            moment().utc().toISOString()
```

Comment:
`moment().utc().toISOString()` and `moment().toISOString()` are same format. 
<img width="326" alt="Screen Shot 2022-08-07 at 17 14 44" src="https://user-images.githubusercontent.com/5715919/183279726-87a8e0da-1d70-43fc-b5bb-6c7ad8921e93.png">

I am doing some changes, let me update it.

Reply:
Updated.